spec/system/claims/support/claims/filtering_claims_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "rails_helper"
+
+RSpec.describe "Filtering claims", type: :system, service: :claims do
+  let(:support_user) { create(:claims_support_user) }
+  let(:school) { create(:claims_school, :claims, name: "School name 1", urn: "1234") }
+  let(:another_school) { create(:claims_school, :claims, name: "School name 1", urn: "1235") }
+  let!(:niot) { create(:claims_provider, :niot) }
+  let!(:best_practice_network) { create(:claims_provider, :best_practice_network) }
+
+  let!(:best_practice_network_claim_1) { create(:claim, :submitted, provider: best_practice_network, school_id: another_school.id, reference: "12345676") }
+  let!(:best_practice_network_claim_2) { create(:claim, :submitted, provider: best_practice_network, school_id: another_school.id, reference: "12345675") }
+
+  before do
+    create(:claim, :submitted, provider: niot, school_id: school.id, reference: "12345678")
+    create(:claim, :submitted, provider: niot, school_id: school.id, reference: "12345677")
+
+    user_exists_in_dfe_sign_in(user: support_user)
+    given_i_sign_in
+  end
+
+  scenario "Support user visits the claims index page and filters the claims and then downloads CSV using the same filters" do
+    given_i_am_on_the_claims_index_page
+    when_i_filter_the_claims_by_provider(best_practice_network)
+    when_the_filter_returns_only_the_best_practice_network_claims([best_practice_network_claim_1, best_practice_network_claim_2], best_practice_network)
```

Comment:
then_the_filter_returns_only_the_best_practice_network_claims
```suggestion
    then_the_filter_returns_only_the_best_practice_network_claims([best_practice_network_claim_1, best_practice_network_claim_2], best_practice_network)
```

Last one I promise 😄